dir.props

```diff
@@ -81,8 +81,13 @@
   <PropertyGroup>
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
+
     <CoreFxExpectedPrerelease>rc4-24131-00</CoreFxExpectedPrerelease>
-    <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(NETStandard\.Library)|(Microsoft\.CSharp)|(Microsoft\.NETCore.*)|(Microsoft\.TargetingPack\.Private\.(CoreCLR|NETNative))|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)(?&lt;!System\.Data\.SqlClient)(?&lt;!System\.IO\.Compression)$</CoreFxVersionsIdentityRegex>
+    <CoreClrExpectedPrerelease>rc4-24131-00</CoreClrExpectedPrerelease>
+    <CoreLibTargetingPackExpectedPrerelease>rc4-24131-00</CoreLibTargetingPackExpectedPrerelease>
```

Comment:
As I mentioned in your other PR could you just merge CoreLibTargetingPackExpectedPrerelease and ExternalExpectedPrerelease into just External, at least until we see a path where they are split.